wordle-web/src/views/WordleView.vue

```diff
@@ -35,21 +37,22 @@ function checkGuess() {
 }
 
 function addChar(letter: Letter) {
+  audio.play()
   game.guess.push(letter.char)
-  guess.value += letter.char
+  game.validWordList = WordsService.validWords(game.guess.text)
 }
 
 function keyPress(event: KeyboardEvent) {
-  console.log(event.key)
   if (event.key === 'Enter') {
     checkGuess()
   } else if (event.key === 'Backspace') {
     guess.value = guess.value.slice(0, -1)
     game.guess.pop()
-    console.log('Back')
   } else if (event.key.length === 1 && event.key !== ' ') {
```

Comment:
This also does not validate what's inside guess.value. Make sure guess.value does not become larger than what should be allowed. It's possible to not even use a local guess reference at all - but that would require a bit more to get it working.